Dockerfile-online

```diff
@@ -0,0 +1,7 @@
+FROM java:openjdk-8-jre-alpine
+
+WORKDIR /home
+
+COPY target/*.jar /home
+
+ENTRYPOINT java -jar *.jar
```

Comment:
Why using wildcards here?

Reply:
Cause we couldn't know the specific jar name directly. If you have any better suggestions, you can come up with them.